pkg/gui.py

```
@@ -190,152 +195,112 @@ def check_status(self, thread):
+            self.body_frame.update()
 
-    def add_to_history(self, item):
+    def add_to_history(self, message_item):
         '''
         adds a message to the main apps window that doesn't expire like the popup
         '''
-        msg = item[0]
-        display_info = item[1]
-        print("GUI 2: ", msg)
+        msg = message_item[0]
+        logging.debug(f"GUI 2: {msg}")
```

Ruff (0.8.0)

Logging statement uses f-string

(G004)